vlp/api/helpers.py

```diff
@@ -1,3 +1,24 @@
+import yt_dlp as youtube_dl
 # Helper functions
 
-# Video Download
+download_directory = 'vlp/youtube-downloads'
```

Comment:
Instead please import BASE_DIR from settings.py and then do `os.path.join(BASE_DIR,'youtube-downloads')`